.github/workflows/doc8.yaml

```diff
@@ -0,0 +1,54 @@
+name: doc8 Pull Request
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  # Triggers the workflow when pull request created and updated
+  pull_request:
+    # Sequence of patterns matched against refs/heads
+    branches:
+      - main
+  # Allows run of this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  check_code_doc8:
+    runs-on: ${{ vars.RUNNER }}
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "pypy-3.10"
```

Comment:
Do we want to be using `pypy` here? This refers to an alternative python interpreter, not the cPython interpreter. I think we probably want `python-version: "3.10"` here instead.